api/environments/identities/models.py

```diff
@@ -38,6 +38,13 @@ class Meta:
     def natural_key(self):
         return self.identifier, self.environment.api_key
 
+    @property
+    def composite_key(self):
+        return f"{self.environment.api_key}_{self.identifier}"
+
+    def get_hash_key(self, use_mv_v2_evaluation: bool = False) -> str:
```

Comment:
I guess here we don't technically need to pass in the `use_mv_v2_evaluation` argument, right? We can just do: 

```python
return self.composite_key if self.environment.use_mv_v2_evaluation else str(self.id)
```